base_layer/core/src/validation/error.rs

```diff
@@ -242,7 +244,9 @@ impl ValidationError {
                 ban_duration: BanPeriod::Long,
             }),
             ValidationError::MergeMineError(e) => e.get_ban_reason(),
-            ValidationError::FatalStorageError(_) | ValidationError::IncorrectNumberOfTimestampsProvided { .. } => None,
+            ValidationError::FatalStorageError(_) |
+            ValidationError::IncorrectNumberOfTimestampsProvided { .. } |
+            ValidationError::Other(_) => None,
```

Comment:
this is wrong, any failure, of the c29 difficulty, should result in a difficulty to low error. And thus a ban

Reply:
resolved